ui/src/components/features/build-panel/components/form-select.tsx

```diff
@@ -32,7 +38,7 @@ export function FormSelect({ form, title ,options, formName, placeholder }: TFor
         <FormItem className="flex flex-col">
           <FormLabel>{title}</FormLabel>
           <FormControl>
-            <Select onValueChange={field.onChange} defaultValue={field.value}>
+            <Select onValueChange={field.onChange} value={field.value}>
```

Comment:
Oh. That was the problem. I feel like I glazed right over this and tried a ton of other things... lol. Thank you Stas!